tests/test_examples.py

```diff
@@ -1,102 +1,104 @@
 import numpy as np
+import pytest
 import flexcode
 import xgboost as xgb
 from flexcode.regression_models import NN, RandomForest, XGBoost, Lasso, CustomModel
 
-
+@pytest.mark.skip(reason="The assertion is meaningless and the test is a duplicate")
```

Comment:
I've marked this test, and the following one as `skip` because the assertion doesn't do anything. And as near as I can tell they are duplicates to unit tests in test_cv_optim.